packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -0,0 +1,368 @@
+import Chevron from '../ui-icons/Chevron';
+import {Divider} from './Divider';
+import {filterDOMProps} from '@react-aria/utils';
+import {mergeStyles} from '../style/runtime';
+import React, {createContext, forwardRef, useContext} from 'react';
+import {size as sizeValue, style} from '../style/spectrum-theme' with { type: 'macro' };
+import {StyleString} from '../style/types';
+import {UnsafeStyles} from './style-utils' with { type: 'macro' };
+import {useDOMRef} from '@react-spectrum/utils';
+import {useLocale} from '@react-aria/i18n';
+import {useSpectrumContextProps} from './useSpectrumContextProps';
+
+
+export interface AccordionGroupProps extends UnsafeStyles, DOMProps, SlotProps {
+  /** The accordion items in the accordion group. */
+  children: React.ReactNode,
+  /** Spectrum-defined styles, returned by the `style()` macro. */
+  styles?: StyleString,
+  /**
+   * The size of the accordion.
+   * @default "M"
+   */
+  size?: 'S' | 'M' | 'L' | 'XL',
+  /**
+   * The amount of space between the accordion items.
+   * @default "regular"
+   */
+  density?: 'compact' | 'regular' | 'spacious',
+  /** Whether the accordion should be displayed with a quiet style. */
+  isQuiet?: boolean,
+  /** Whether the accordion should be disabled. */
+  isDisabled?: boolean
+}
+
+const groupStyles = style({
+  display: 'flex',
+  flexDirection: 'column',
+  minWidth: sizeValue(200)
+});
+
+export const AccordionGroupContext = createContext<ContextValue<AccordionGroupProps, DOMRefValue<HTMLDivElement>>>(null);
+
+let InternalAccordionGroupContext = createContext<Pick<AccordionGroupProps, 'size'| 'isQuiet'| 'density' | 'isDisabled'>>({});
+
+function AccordionGroup(props: AccordionGroupProps, ref: DOMRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, AccordionGroupContext);
+  let domRef = useDOMRef(ref);
+  let {
+    UNSAFE_style,
+    UNSAFE_className = '',
+    size = 'M',
+    density = 'regular',
+    isQuiet,
+    isDisabled,
+    ...otherProps
+  } = props;
+  const domProps = filterDOMProps(otherProps);
+  return (
+    <Provider
+      values={[
+        [InternalAccordionGroupContext, {size, isQuiet, density, isDisabled}]
+      ]}>
+      <div
+        {...domProps}
+        ref={domRef}
+        style={UNSAFE_style}
+        className={(UNSAFE_className ?? '') + mergeStyles(groupStyles, props.styles)}>
+        {props.children}
+      </div>
+    </Provider>
+  );
+}
+
+/**
+ * A accordion group is a container for multiple accordion items.
+ */
+let _AccordionGroup = /*#__PURE__*/ (forwardRef as forwardRefType)(AccordionGroup);
+export {_AccordionGroup as AccordionGroup};
+
+const itemStyles = style({
+  color: 'heading',
+  padding: '[2px]'
```

Comment:
this never changes? won't spacing be static, so `2` should be fine?
I think devon recently added support for 2 in a PR, don't recall if it was in cards though, might have a check there

Reply:
Removed this after adding negative outline offset